openeo_processes_dask/process_implementations/cubes/reduce.py

```diff
@@ -54,9 +54,8 @@ def reduce_spatial(
     spatial_dims = data.openeo.spatial_dims if data.openeo.spatial_dims else None
     return data.reduce(
         reducer,
-        dimension=spatial_dims,
+        dim=spatial_dims,
         keep_attrs=True,
-        context=context,
```

Comment:
Why is context being dropped here? To my knowledge this is passed downstream to the reducer function which might be using it no?

Reply:
That's true, it was a mistake, fixed now.

Reply:
I think you can remove the context, as it gets passed to the subprocess in the `named_parameters`, then the tests should pass again :)